ui/src/pages/tests/tests.tsx

```diff
@@ -60,9 +51,9 @@ export default function Tests() {
       <div>
         <br />
         Link 1.:{' '}
-        <a href="" className="link_primary">
+        <Link to={paths.REGISTER} className="link_primary">
           Zarejestruj się
```

Comment:
myślę ze jeszcze warto dodać i18n, mimo że nie mamy w planach dodawania więcej jezyków niż polski, to warto zabezpieczyć się na przyszłość, a nie jest to jakiś duży koszt, tymbardziej na początku projektu